environments/common/inventory/group_vars/all/openhpc.yml

```diff
@@ -57,6 +57,7 @@ openhpc_config_default:
     - enable_configless
   TaskPlugin: task/cgroup,task/affinity
   ReturnToService: 2 # workaround for templating bug TODO: Remove once on stackhpc.openhpc v1.2.0
+  TopologyPlugin: topology/tree
```

Comment:
What happens if you have nothing in `topology` group and this is enabled? Does it need to be guarded/nulled out somehow if that is the case, or is it OK?